applications/serial_lte_modem/src/twi/slm_at_twi.c

```diff
@@ -0,0 +1,404 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <logging/log.h>
+#include <zephyr.h>
+#include <drivers/i2c.h>
+#include <stdio.h>
+#include "slm_util.h"
+#include "slm_at_twi.h"
+
+LOG_MODULE_REGISTER(slm_twi, CONFIG_SLM_LOG_LEVEL);
+
+#define TWI_MAX_INSTANCE	4
+#define TWI_ADDR_LEN		2
+#define TWI_DATA_LEN		255
+
+/**@brief List of supported AT commands. */
+enum slm_twi_at_cmd_type {
+	AT_TWI_OPEN,
+	AT_TWI_WRITE,
+	AT_TWI_READ,
+	AT_TWI_WRITE_READ,
+	AT_TWI_MAX
+};
```

Comment:
No need of this enum any more after SLM restructure.
But you can reuse it as OP code if you change to AT#XTWI=< op >, ...

Reply:
Yes. These OPs are removed.